.github/workflows/ci.yml

```diff
@@ -42,4 +46,4 @@ jobs:
     - name: Spelling
```

Comment:
Hmm. I'm not sure about spell check failure being a formal part of the CI pipeline. I can easily see that being a pain in the butt. I'm guessing running it without `-error` means we just get the check notifications below, but the build still passes?

Reply:
yep. I removed the `-error` because I didn't want the build the fail because of a misspelling.